web/src/classic/components/molecules/Visualizer/WidgetAlignSystem/utils.ts

```diff
@@ -0,0 +1,31 @@
+import { isBuiltinWidget } from "../Widget/builtin";
+
+import { Widget, WidgetZone } from "./hooks";
+
+const sections = ["left", "center", "right"] as const;
+const areas = ["top", "middle", "bottom"] as const;
+
+export const filterSections = (
+  zone?: WidgetZone,
+  invisibleWidgetIDs?: string[],
+  cb?: (s: (typeof sections)[number]) => void,
+) => {
+  return sections.filter(
+    s =>
+      areas.filter(a => zone?.[s]?.[a]?.widgets?.find(w => !invisibleWidgetIDs?.includes(w.id)))
```

Comment:
Same with [this](https://github.com/reearth/reearth/pull/839/files#r1408956105)

Reply:
Updated. Thank you!